satellite/payments/stripecoinpayments/clearing.go

```diff
@@ -19,25 +20,29 @@ var ErrClearing = errs.Class("stripecoinpayments clearing error")
 // Clearing runs process of reconciling transactions deposits,
 // customer balance, invoices and usages.
 type Clearing struct {
```

Comment:
Add architecture annotations to mark it as a Chore.

Also if there are no-naming conflicts, you can also rename it to `Chore`.